### DISCUSSION TITLE
How could I switch from having the Github Education Program from one GitHub account to another?

### DISCUSSION THREAD

**Discussion by YousufSSyed** [community/community#46939] [GitHub Education]
I signed up for it with a new account using a school email, but I change my mind and want to have the education program benefit under this one.

**Comment by jge162** [comment]
Hello @YousufSSyed that is something you have to discuss with [education support](https://support.github.com/request/education?tags=hubberfy_education). Use other support option.

**Reply by samus-aran** [reply]
😄 We wouldn't recommend this type of inquiry be directed to support but thank you for making the suggestion!

**Reply by jge162** [reply]
I will remember that. Thank you too. Have a great Friday.

**Comment by samus-aran** [comment]
Hi @YousufSSyed  :wave:!

It’s not possible to transfer Global Campus benefits over from one GitHub account to another.

However, you are able to merge your accounts, but the account with the benefits would be the main account. You can add your non-academic email from the previous account to the account containing the academic email. If you desire you can then rename this account once merged. For more details check out out the [merging of your personal accounts](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-personal-account-on-github/managing-your-personal-account/merging-multiple-personal-accounts) documentation.
